tox.ini

```diff
@@ -1,10 +1,12 @@
 [tox]
-envlist = py27, pypy
+envlist = py27, pypy, py34
 
 [testenv]
 deps = 
     pytest
     pytest-capturelog
+    tzlocal
+    configobj
```

Comment:
Shouldn't be necessary, `khal` gets installed as pkg (including dependencies as specified in `setup.py`)